libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp

```diff
@@ -69,23 +69,23 @@ static QString TurnCommandType(const RouteDescription::DirectionDescriptionRef&
 static QString ShortTurnCommand(const RouteDescription::DirectionDescriptionRef& directionDescription)
 {
   if (!directionDescription){
-    return osmscout::RouteDescriptionBuilder::tr("Turn");
```

Comment:
I am worry that qt translation is not working with full namespace :-/ Will need to check

Reply:
OK, there ws a warning regarding namespaces, so I tried...

I'm still struggeling with glm+meson+OS X. Looks like I needed to make some updates, to get better C++20 support.

Reply:
See the gcc and cmake builds (still contain such wearnings)

Reply:
this small comment fix lupdate: https://github.com/Framstag/libosmscout/pull/1598